.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-windows`
Validate Update-Help from the Web for one PowerShell module..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```

Reply:
It seems to be a sporadic failure, will issue an empty commit to retry